AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 ukrainianwall.com##.privacy-policy
+thgss.com###cookiesUse
```

Comment:
I can't download a file without accepting cookies.
1. Exchange this rule to `thgss.com#%#//scriptlet("set-cookie", "acceptUsingCookies", "yes")`.
2. Add `thgss.com#@##acceptCookies` with task comment to the `cookies_whitelist.txt`